src/features/defi/providers/thorchain-savers/components/ThorchainSaversManager/Overview/ThorchainSaversOverview.tsx

```diff
@@ -284,8 +310,16 @@ export const ThorchainSaversOverview: React.FC<OverviewProps> = ({
         label: 'common.withdraw',
         icon: <ArrowDownIcon />,
         action: DefiAction.Withdraw,
-        isDisabled: hasPendingTxs || hasPendingQueries || isDisabledWithdrawals,
+        isDisabled:
+          hasPendingTxs ||
+          hasPendingQueries ||
+          isDisabledWithdrawals ||
+          !isRunepoolWithdrawUnlocked,
```

Comment:
blockiing: we will want to check for `isRunePool` in addition to this now that the sig of `isRunepoolWithdrawUnlocked` changed, else withdraw of regular savers will be borked.

```suggestion
          (isrunePool && !isRunepoolWithdrawUnlocked),
```

<img width="560" alt="image" src="https://github.com/user-attachments/assets/293e7b6e-3494-4fe9-bf5f-17feaa494353">

Reply:
Nice catch... fixed in [231aabd](https://github.com/shapeshift/web/pull/7472/commits/231aabd2822d7bf58613c7fd32ba5873c67846a8)